app/views/dashboard/index.html.erb

```diff
@@ -14,14 +14,20 @@
   <div class="m-4">
     Main Content
   </div>
-  <ul class="flex flex-row flex-wrap gap-4 m-4">
-    <% (1.. 30).to_a.sample.times do |n| %>
-      <a href="/dialog" data-turbo-frame="dialog" class="h-40 w-40 border-2 bg-slate-400/80 cursor-pointer">&nbsp;</a>
+  <ul id="main-content-list" class="flex flex-row flex-wrap gap-4 m-4" data-turbo-permanent>
+    <% (99.. 200).to_a.sample.times do |n| %>
```

Comment:
Turbo 8 "supposedly" preserves scroll position, but honestly it's a bit finicky and goes back to the top 1/10 times, and moves some slight amount 7/10 times.